src/components/textarea/textarea.stories.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import {
+  boolean, number, text, select
+} from '@storybook/addon-knobs';
+import { State, Store } from '@sambego/storybook-state';
+import Textarea from './textarea';
+import OptionsHelper from '../../utils/helpers/options-helper';
+import notes from './notes.md';
+
+const store = new Store({
+  value: ''
+});
+
+const handleChange = ({ target: { value } }) => {
+  store.set({ value });
+};
+
+const rangeOptions = {
+  range: true,
+  min: 0,
+  max: 300,
+  setp: 1
+};
+
+storiesOf('Textarea', module).add(
+  'default',
+  () => {
+    const { alignBinary } = OptionsHelper;
+
+    const warnOverLimit = boolean('warnOverLimit', false);
+    const labelInline = boolean('labelInline', false);
+    const fieldHelpInline = boolean('fieldHelpInline', false);
+    const expandable = boolean('expandable', false);
+    const characterLimit = number('characterLimit', 0, rangeOptions);
+    const enforceCharacterLimit = characterLimit ? boolean('enforceCharacterLimit', true) : undefined;
```

Comment:
Can we take the default value from the component? I.e. `Textarea.defaultProps. enforceCharacterLimit `